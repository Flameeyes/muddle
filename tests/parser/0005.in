How about some escaping?

$${I wonder if this works?}
And this should be literal: $$
How about \${ ?
And really adventurously: ${fn:echoargs("a\"b", c\,d)}
