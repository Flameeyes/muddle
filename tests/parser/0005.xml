<?xml version="1.0" ?>

<top>
  <my-name-is>And</my-name-is>
</top>
