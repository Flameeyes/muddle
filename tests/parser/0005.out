Multiple parameters?

Hello, I am abcde,f.
